protostar/cheatable_starknet/callable_hint_locals/print_hint_local.py

```diff
@@ -0,0 +1,20 @@
+from typing import Callable
+
+from protostar.cheatable_starknet.controllers.io import IOController
+from protostar.starknet.data_transformer import CairoData
+
+from .callable_hint_local import CallableHintLocal
+
+
+class PrintHintLocal(CallableHintLocal):
+    @property
+    def name(self) -> str:
+        return "protostar_print"
+
+    def _build(
+        self,
+    ) -> Callable[[CairoData], None]:
+        return self.protostar_print
+
+    def protostar_print(self, data: CairoData) -> None:
+        IOController.protostar_print(data=data)
```

Comment:
I think it'd be better to inject IOController like we do with all other controllers in case we change something in it and protostar_print is no longer static/new methods are added.

Reply:
Why? IMO we should do dependency injection on demand, not because it might me useful in the future

Reply:
Just for future proofing + it's consistent with the rest of the hint locals.